instance/tests/integration/integration_instance.py

```diff
@@ -279,6 +278,7 @@ def test_spawn_appserver(self):
         """
         OpenEdXInstanceFactory(
             name='Integration - test_spawn_appserver',
+            configuration_playbook_name='playbooks/opencraft_integration.yml',
```

Comment:
@giovannicimolin, wouldn't it be better to mock the `get_base_playbook_name()` method to return this value for all the tests instead of repeating this argument in every test for every case where we do not expect the playbook to fail?

Reply:
Done!